src/components/Modal/ModalCloseButton.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+
+import { useLocale } from '../../i18n'
+import { ExternalStyles } from '../../styles'
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import { Tooltip } from '../Tooltip'
+
+interface ModalCloseButton {
+  onClose(): any
```

Comment:
This isn't a good practice, you always need to define a type
If you don't do this, anyon can pass a string or number to `onClose`